src/form/textarea/Textarea.tsx

```diff
@@ -0,0 +1,137 @@
+import "./_textarea.scss";
+
+import React, {useState} from "react";
+import classNames from "classnames";
+import TextareaAutosize, {TextareaAutosizeProps} from "react-textarea-autosize";
+
+import {KEYBOARD_EVENT_KEY} from "../../core/utils/keyboardEventConstants";
+
+type TextareaProps = Omit<
+  React.TextareaHTMLAttributes<HTMLTextAreaElement>,
+  "name" | "onChange" | "className"
+> & {
+  name: string;
+  onChange: React.ReactEventHandler<HTMLTextAreaElement>;
+  value?: string;
+  testid?: string;
+  isDisabled?: boolean;
+  customClassNames?: {
+    container?: string;
+    textarea?: string;
+  };
+  style?: React.CSSProperties;
+  autoSizeProps?: TextareaAutosizeProps;
+  onJustEnterPressed?: () => void;
+  onShiftEnter?: () => void;
```

Comment:
I think both are OK, but I usually prefer `VoidFunction` (a pre-defined type) instead of `() => void`. They represent the same thing, but it is easier to type `VoidFunction` 😄

Reply:
Yes, looks good. So, should we refactor all of the `() => void` usage to `VoidFunction`? It's just for consistency.

Reply:
I don't think we should, if `() => void` is more common in this codebase, just wanted to let you know about this option. It can stay like this 👍

Reply:
Alright, I'm reverting the related change. Thank you for your comments! 🎉